mkosi-initrd-posttrans

```diff
@@ -36,16 +36,27 @@ initrd_regenerate() {
 }
 
 initrd_regenerate_all() {
+	local d
+
 	# FIXME: mkosi-initrd does not provide anything like --regenerate-all yet
 	for d in /lib/modules/*; do
 		[ -d "$d" ] || continue
 		kver=${d##*/}
-		if "$MKOSI_INITRD" --kernel-version "$kver" -O "$MKOSI_INITRD_STAGING_DIR" -o "initrd-$kver" \
-			&& chmod 600 "$MKOSI_INITRD_STAGING_DIR/initrd-$kver"; then
-			# The staging dir will contain: initrd-<kver> -> initrd-<kver>.cpio.zst
-			cp --reflink=auto "$MKOSI_INITRD_STAGING_DIR/initrd-$kver" "/boot/initrd-$kver"
-			rm -f "$MKOSI_INITRD_STAGING_DIR/initrd-$kver"*
-		fi
+
```

Comment:
why don't you just pull the "cp --reflink" command in to the "if" statement (with `&&`) and add 
```
else
     return $?
fi
```
?

Reply:
Ok, changed.